### DISCUSSION TITLE
Are my apps updating?

### DISCUSSION THREAD

**Discussion by thepwds** [FossifyOrg/General-Discussion#26] [General]
I got the Simple Mobile Tools apps from F-Droid. The apps have no subscription stuff at all, and the apps seem to be removed from the F-Droid repositories. Is it safe to keep these apps on my phone for the time being until these Fossify apps start to be on F-Droid? Also i think they should change the main color from orange to green.

**Comment by Aga-C** [comment]
Don't worry, F-Droid versions are safe.

**Reply by thepwds** [reply]
Awesome, thanks for the confirmation

**Comment by naveensingh** [comment]
> i think they should change the main color from orange to green.

we'll be doing that, thank you.

**Comment by naveensingh** [comment]
Let's keep this discussion open for now. There will probably be more people asking this.

**Comment by inson1** [comment]
@bplx "the apps seem to be removed from the F-Droid repositories" Not removed, only have disabled AUM (AutoUpdateMode) https://gitlab.com/fdroid/fdroiddata/-/merge_requests/14135
